react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -96,7 +105,7 @@ class Thumbnail extends Component {
                 <ParticipantView
                     avatarSize = { AVATAR_SIZE }
                     participantId = { participantId }
-                    tintEnabled = { participantInLargeVideo }
+                    tintEnabled = { participantInLargeVideo && !disablePin }
```

Comment:
Why is this needed? `participantInLargeVideo` will always be false in tile view anyway. Am I missing something?

Reply:
Nothing clears large video state when switching to tile view. Large video updates are allowed to proceed on web because electron always-on-top needs to know who is on large video. I could try to find a way to clear the state, but maybe on mobile only.

Reply:
> Edit: at the very, very least, I probably shouldn't re-use the disablePin prop to prevent tinting.

👍